docs/development/local_setup.md

```diff
@@ -61,7 +61,7 @@ $ kubectl apply -f kubernetes/crds.yaml
 
 ### Get started
 
-Create a `dev` directory and copy the Kubeconfig of the kubernetes cluster used for development purposes to `dev/kubeconfig.yaml`.
+Create a `dev` directory and copy the Kubeconfig of the both kubernetes clusters used for development purposes to `dev/kubeconfig.yaml` and `dev/kubeconfig-seed.yaml`.
```

Comment:
We need to make the documentation more understandable. Also `make dev` fails to start right now when there is no kubeconfig-seed, we should make it more robust ?

Reply:
Agreed, we must be able to start as well when no "management kubeconfig" has been provided.

Reply:
We can add, how to manipulate 2 kubeconfigs and have it inside the documentation in detail.  `make dev` would have failed because I was expecting 2 kubeconfigs under `dev/`, may be I will make changes in the make-rules to work fine when only one kubeconfig is available.